### PR TITLE
chore: upgrade to csi-proxy v1.0.0 GA binary

### PR DESCRIPTION
**Pull request by andyzhangx**
csi-proxy v1.0.0 GA binary is compatible with v0.2.2 beta interface.

### CODE REVIEW

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 1]
```diff
@@ -93,7 +93,7 @@ $global:map = @{
         $defaultContainerdPackageUrl
     );
     "c:\akse-cache\csi-proxy\"    = @(
-        "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz"
+        "https://acs-mirror.azureedge.net/csi-proxy/v1.0.0/binaries/csi-proxy-v1.0.0.tar.gz"
```
Comment (andyliuliming) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:96]: @AbelHu we do not store two versions in windows image?

Comment (AbelHu) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:96]: Agree. cc @mainred we need to cache this in new VHD with 8B

Comment (andyzhangx) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:97]: it's done

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 2]
```diff
@@ -94,6 +94,7 @@ $global:map = @{
     );
     "c:\akse-cache\csi-proxy\"    = @(
         "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz"
```
Comment (AbelHu) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:96]: Add a `,`

**Review by AbelHu — APPROVED** [review]

**Review by mainred — APPROVED** [review]
I will merge the PR without waiting for the check-in to give more time for vhd building.

**Pull request merged**